jpsreport/Utest/Methods/runtest_Method_A.py

```diff
@@ -1,53 +0,0 @@
-#!/usr/bin/env python
```

Comment:
why did you delete these files? Or did you just move them?

Reply:
The original tests created one tests for each outputfile to validated. I restructured this so that all generated outputfiles (based on one inifile) are validated/compared with a reference directory in one test.
Also, statistics functions have been used to check/assess the detected difference in outputfiles. I removed this.
I deleted the old tests as there were major changed. Should we keep the old files?